### ISSUE TITLE
[Feature Request]  Ability to create new SSH Connections allowing for input of the username and password

### ISSUE THREAD

**Issue by IGIraffeT** [Eugeny/tabby#4608]
Hey! Love the application so far for use at home. For use at work however, I happen to connect to a lot of different systems that require my username/password to be entered. I know there's a 'Default for SSH' pane that's configurable, however I don't think it quite works as I had anticipated:

![image](https://user-images.githubusercontent.com/35666729/133640387-a8096210-c2a7-4f17-82af-e8cd55a268ab.png)

Would it be possible to incorporate similar functionality to something like PuTTY or MobaXTerm, where upon entering the hostname of a system, you're prompted to enter a username and password to log into the system? Thanks!

**Comment by mhassankhan90** [comment]
Hi, I recently came across this application. I am also facing this issue. SSH profiles where username is provided works fine. But sometimes, we need to login with different ID and creating a profile for each ID is not intuitive. I tried to explore things on my own. I tried the "Interactive" option in authentication method but it asks for password only. There is no option to enter username.
I think if this option is added, then this is going to be my default client.
